docs/reference/configurations.md

```diff
@@ -0,0 +1,82 @@
+# Configurations
+
+- _blog_hostname_: Hostname for accessing WordPress, if ingress relation is active. Defaults to the
```

Comment:
Same here, put the config names as they will be used in cmd line, that is, as they appear in config.yaml